Mono.Cecil.PE/ImageWriter.cs

```diff
@@ -719,7 +719,7 @@ void BuildTextMap ()
 					entry.Directory = directory;
 
 					data_len += entry.Data.Length;
-					data_address += data_len;
+					data_address += entry.Data.Length;
```

Comment:
this is more index than address, right?

Reply:
It also RVA - which is "Address" as well.